spark/src/main/resources/interpreter-setting.json

```diff
@@ -77,6 +77,12 @@
         "propertyName": "zeppelin.spark.maxResult",
         "defaultValue": "1000",
         "description": "Max number of SparkSQL result to display."
+      },
+      "zeppelin.spark.importImplicit": {
+        "envName": "ZEPPELIN_SPARK_IMPORTIMPLICIT",
+        "propertyName": "zeppelin.spark.importImplicit",
```

Comment:
Yeah, you are right, also it does sound a little bit too broad. I am thinking about `zeppelin.spark.sql.importImplicit`, or `zeppelin.spark.importSQLImplicit`. What do you think?